tests/acceptance/SaveNewsletterAsDraftCest.php

```diff
@@ -30,14 +30,15 @@ function saveStandardNewsletterAsDraft(\AcceptanceTester $I) {
     $I->seeInCurrentUrl('mailpoet-newsletter-editor');
     $I->fillField($title_element, $newsletter_title);
     $I->click('Next');
+
     // step 4 - Choose list and send
     $send_form_element = '[data-automation-id="newsletter_send_form"]';
     $I->waitForElement($send_form_element);
     $I->seeInCurrentUrl('mailpoet-newsletters#/send/');
     $I->selectOptionInSelect2('WordPress Users');
 
     $I->click('Save as draft and close');
-    $I->waitForText($newsletter_title, 5, '[data-automation-id^="listing_item_"]');
+    $I->waitForText($newsletter_title, 30);
```

Comment:
This is kind of in conflict with what Alex was working on. Basically the idea is to never use "magic" timeouts in tests and leave it to the default 10s. If we need longer timeout in some, then there would be a single constant for that. But here - is really 10s not enough?

Reply:
Removed timeout completely